src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Level,
+  LevelItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+  Title,
+} from "@patternfly/react-core";
+import type { AlertProps } from "@rhoas/app-services-ui-shared";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+
+export type SettingsProps = {
+  showAlert: AlertProps[];
+  isChecked: boolean;
+  isModalOpen: boolean;
+  onChange: () => void;
+  onDisable: () => void;
+  onClose: () => void;
+  closeAlertAction: (key: string | undefined) => void;
+  isLoading: boolean;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  showAlert,
+  isChecked,
+  onChange,
+  isModalOpen,
+  onDisable,
+  onClose,
+  closeAlertAction,
+  isLoading,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection isWidthLimited isCenterAligned>
+          <Card>
+            <CardBody>
+              <Level>
+                <LevelItem>
+                  <Flex>
+                    <FlexItem spacer={{ default: "spacerNone" }}>
```

Comment:
Using `default: "spacerNone"` here results in no space between the label and the value, which looks odd:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/21063328/181785514-768f9ddb-c68a-4b4f-99c1-93c420f2e710.png">

Use the `sm` spacer instead. That will add some space as shown below:
<img width="307" alt="image" src="https://user-images.githubusercontent.com/21063328/181785771-15121924-7684-4e43-8103-96ba4b4fd9f3.png">